docs/news.rst

```diff
@@ -911,6 +911,20 @@ Subparsers can be listed with ``--list-subparser``:
 Changes to the tags file format
 ---------------------------------------------------------------------
 
+.. NOT REVIEWED YET
+
+version 3 format for handling meta characters in tag entry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To make allow to record a name including a tab character to a tag file,
+the tags file format is extended, and we call the new file format version 3.
+
+You can use ``--format=2`` to keep the compatibility between the output of
+Exuberant-ctgs. However, the old file format cannot record a name including
+a tab character.
```

Comment:
*“You can use ``--format=2`` to keep compatibility with the output of Exuberant-CTags. However, this version of the format cannot record a names including tab characters.”*

Reply:
*“You can use `--format=2` to keep compatibility with the output of Exuberant-CTags; however, this version of the format cannot record names which include tab characters.”*